server/apps/main/mod_message.txt

```diff
@@ -0,0 +1,22 @@
+Your story on AutSPACEs has been moderated
+Dear AutSPACEs community member,
+
+Thank you for submitting the following story to the AutSPACEs platform:
+
+"{title}"
+
+The moderation status of your story has changed. To see the latest information
+for your story, please visit the AutSPACEs site and navigate to the "My
+Stories" page.
+
+Alternatively, to go directly to the story, select this link:
+
+{story}
+
+You can disable these alerts at any time by visiting your profile page:
+
+{profile}
```

Comment:
I wonder if we should put this a the very end as a kind of email signature (I know the OH stuff wraps it's own signature around it, but maybe worth to put that further down?)